playwright/Customizations/Compliance.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { expect } from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+
+import { test } from '../fixtures/customizations';
+import { ensureAuthenticated } from '../helpers/login';
+import {
+  fillInImageOutput,
+  ibFrame,
+  navigateToLandingPage,
+} from '../helpers/navHelpers';
+import {
+  createBlueprint,
+  deleteBlueprint,
+  fillInDetails,
+  registerLater,
+} from '../helpers/wizardHelpers';
+
+test('Create a blueprint with Compliance policy selected', async ({
+  page,
+  cleanup,
+}) => {
+  const blueprintName = 'test-' + uuidv4();
+
+  // Delete the blueprint after the run fixture
+  await cleanup.add(() => deleteBlueprint(page, blueprintName));
+
+  await ensureAuthenticated(page);
+
+  // Navigate to IB landing page and get the frame
+  await navigateToLandingPage(page);
+  const frame = await ibFrame(page);
+
+  await test.step('Navigate to optional steps in Wizard', async () => {
+    await fillInImageOutput(frame);
+    await registerLater(frame);
+  });
+
+  await test.step('Select Compliance and choose a policy if available', async () => {
+    await frame.getByRole('button', { name: 'Security' }).nth(1).click();
+
+    await frame
+      .getByRole('radio', { name: 'Use a custom Compliance policy' })
+      .check();
+
+    const toggle = frame.locator(
+      '[data-ouia-component-id="compliancePolicySelect"]',
+    );
+    await expect(toggle).toBeEnabled();
+    await toggle.click();
+
+    const policyOption = frame.getByRole('option', {
+      name: /placehollder/i,
```

Comment:
```suggestion
      name: /placeholder/i,
```

Just to check why we're not selecting a specific policy here, I assume:
- there would be an overlap with compliance boot tests which already tests the integration
- and since it's a fresh user it would probably be a pain to setup any compliance policies before we'd be even able to pick any of them

is that right?

Reply:
Aha! Mea culpa, please ignore then. Mock-y sounding mock sounds fine to me :sweat_smile:

Reply:
alrighty, thanks!